src/training/grammar.ts

```diff
@@ -8,25 +8,91 @@ export const multiply = (a: number, b: number): number => {
 };
 
 /* 否定・or・andをそれぞれ記述してください */
+function branchProcess(x: number, y: number) {
```

Comment:
できればアロー関数を使うようにしてください。
(PJ内ではアロー関数を使うことが多いので)
```suggestion
const branchProcess = (x: number, y: number): void => {
```

Reply:
アロー関数で宣言しなおしました。